src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/ActionMetric.kt

```diff
@@ -77,17 +77,29 @@ data class ActionMetric @Deprecated("Use ActionMetric.Builder instead.") constru
 
     class Builder(
         private val label: String,
-        private val result: ActionResult,
-        private val duration: Duration,
+        private var result: ActionResult,
+        private var duration: Duration,
         private val start: Instant
     ) {
         private var observation: JsonObject? = null
         private var drilldown: RecordedPerformanceEntries? = null
         private var virtualUser: UUID = UUID.randomUUID()
 
+
+        constructor(label: String, virtualUser: UUID, start: Instant) : this(
```

Comment:
This is the "dangerous defaults ad hoc constructor" mentioned later

Reply:
I overlooked that. Thanks.